index.js

```diff
@@ -55,6 +55,7 @@ import {
   executeParallelGradleProperties,
   extractJarArchive,
   frameworksList,
+  generatePixiLockFile,
```

Comment:
We should have repo tests for pixi with type python which would have caught this